src/test/java/no/sikt/nva/pubchannels/handler/fetch/journal/FetchJournalByIdentifierAndYearHandlerTest.java

```diff
@@ -288,7 +291,8 @@ void shouldReturnRedirectWhenChannelRegistryReturnsRedirect() throws IOException
         var response = GatewayResponse.fromOutputStream(output, HttpResponse.class);
         assertEquals(HttpURLConnection.HTTP_MOVED_PERM, response.getStatusCode());
         var expectedLocation = constructExpectedLocation(newIdentifier, year);
-        assertEquals(expectedLocation, response.getHeaders().get("Location"));
+        assertEquals(expectedLocation, response.getHeaders().get(LOCATION));
+        assertEquals(WILD_CARD, response.getHeaders().get(ACCESS_CONTROL_ALLOW_ORIGIN));
```

Comment:
This is the actual bug being fixed with https://github.com/BIBSYSDEV/nva-commons/commit/74109eb64e9d2df3e884d3096f37158171a21c68